lightningjs-embed.js

```diff
@@ -161,13 +158,26 @@ window.lightningjs || (function(modules){
                         innerFrame[srcAttr] = domainSrc + "void(0);"
                     }
 
-                    // Set the HTML of the iframe. In IE 6, the document.domain from the iframe src hasn't had time to 
-                    // "settle", so trying to access the contentDocument will throw an error. Luckily, in IE 7 we can 
+                    var loadScript = function(e) {
+                        var iframeDocument = e.currentTarget.contentDocument;
+                        var head = iframeDocument.getElementsByTagName('head')[0];
+                        var script = iframeDocument.createElement('script');
+                        script.setAttribute(srcAttr, internalModule.l);
+                        head.appendChild(script);
+                    };
+
+                    // Set the HTML of the iframe. In IE 6, the document.domain from the iframe src hasn't had time to
+                    // "settle", so trying to access the contentDocument will throw an error. Luckily, in IE 7 we can
                     // finish writing the html with the iframe src without preventing the page from onloading
                     try {
                         var frameDocument = innerFrame[contentWindow][documentString];
                         frameDocument.write(buildInnerFrameHtml());
-                        frameDocument.close()
+                        frameDocument.close();
+                        if (innerFrame.addEventListener) {
```

Comment:
is this the right check?

Reply:
yep I check if the function exist, if not we are in < IE9 and should use attachEvent